frontend/src/metabase/nav/components/search/SearchButton/SearchButton.tsx

```diff
@@ -0,0 +1,24 @@
+import { useKBar, VisualState } from "kbar";
+import { useCallback } from "react";
+import { t } from "ttag";
+
+import { isMac } from "metabase/lib/browser";
+import { Button, Icon, Tooltip } from "metabase/ui";
+
+const METAKEY = isMac() ? "⌘" : "Ctrl";
```

Comment:
i've for sure seen this in a few places at this point. i think it should be a util somewhere.